README.md

```diff
@@ -1,112 +1,51 @@
+  <a href="https://packagist.org/packages/contributte/forms-wizard"><img src="https://badgen.net/packagist/v/contributte/forms-wizard"></a>
+</p>
+<p align=center>
+  <a href="https://packagist.org/packages/contributte/forms-wizard"><img src="https://badgen.net/packagist/php/contributte/forms-wizard"></a>
+  <a href="https://github.com/contributte/forms-wizard/LICENSE"><img src="https://badgen.net/github/license/contributte/forms-wizard"></a>
+  <a href="https://bit.ly/ctteg"><img src="https://badgen.net/badge/support/gitter/cyan"></a>
+  <a href="https://bit.ly/cttfo"><img src="https://badgen.net/badge/support/forum/yellow"></a>
+  <a href="https://contributte.org/partners.html"><img src="https://badgen.net/badge/sponsor/donations/F96854"></a>
+</p>
 
-**Composer**
-```
-composer require contributte/forms-wizard
-```
-
-**Config**
-
-```yaml
-extensions:
-    - Contributte\FormWizard\DI\WizardExtension
-```
-
-# Usage
-
-## Component
-
-```php
-
-use Nette\Application\UI\Form;
-
-class Wizard extends Contributte\FormWizard\Wizard {
-
-    protected function finish(): void 
-    {
-        $values = $this->getValues();
-    }
+<p align=center>
+Website 🚀 <a href="https://contributte.org">contributte.org</a> | Contact 👨🏻‍💻 <a href="https://f3l1x.io">f3l1x.io</a> | Twitter 🐦 <a href="https://twitter.com/contributte">@contributte</a>
+</p>
 
-    protected function createStep1(): Form 
-    {
-        $form = $this->createForm();
+## Usage
 
-        $form->addText('name', 'User name')
-            ->setRequired();
+To install latest version of `contributte/forms-wizard` use [Composer](https://getcomposer.com).
 
-        $form->addSubmit(self::NEXT_SUBMIT_NAME, 'Next');
-
-        return $form;
-    }
-
-    protected function createStep2(): Form 
-    {
-        $form = $this->createForm();
-
-        $form->addText('email', 'Email')
-            ->setRequired();
-
-        $form->addSubmit(self::PREV_SUBMIT_NAME, 'Back');
-        $form->addSubmit(self::FINISH_SUBMIT_NAME, 'Register');
-
-        return $form;
-    }
-}
 ```
-
-```yaml
-services:
-    - Wizard
+composer require contributte/forms-wizard
 ```
 
-## Presenter
-
-```php
+## Documentation
 
-final class HomepagePresenter extends Nette\Application\UI\Presenter {
+For details on how to use this package, check out our [documentation](.docs).
 
-    /** @var Wizard @inject */
-    public $wizard;
-    
-    public function handleChangeStep(int $step): void 
-    {    
-        $this['wizard']->setStep($step);
-        
-        $this->redirect('wizard'); // Optional, hides parameter from URL
-    }
 
-    protected function createComponentWizard(): Wizard 
-    {
-        return $this->wizard;
-    }
+## Versions
 
-}
+| State       | Version     | Branch   | PHP      |
+|-------------|-------------|----------|----------|
+| dev         | `2.1.x-dev` | `master` | `^7.1`   |
```

Comment:
@f3l1x  there are nonsenses in versions in composer. Could you fix it?

Reply:
Yep, I will